p2p/host/routed/routed.go

```diff
@@ -28,15 +27,12 @@ const AddressTTL = time.Second * 10
 // This allows the Host to find the addresses for peers when
 // it does not have them.
 type RoutedHost struct {
-	host  host.Host // embedded other host.
-	route Routing
+	host.Host
+	route routing.PeerRouting
 }
 
-type Routing interface {
-	FindPeer(context.Context, peer.ID) (peer.AddrInfo, error)
-}
-
```

Comment:
Can you deprecate this with a comment saying use `routing.PeerRouting`

Reply:
Done.